.github/workflows/ansible-lint.yml

```diff
@@ -22,7 +22,14 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Set ANSIBLE_ROLES_PATH
-      run: echo "ANSIBLE_ROLES_PATH=\"$(find ./projects -type d -name roles -print | tr '\n' ':')$(ls roles/* -d | tr '\n' ':')\"" >> $GITHUB_ENV
```

Comment:
There is no need to get each role path, we only need the 'roles' folder where they all are, and this path needs to be absolute. Also we should preserve the previous paths.